sample/Kmd.Logic.Cvr.Client.Sample/Program.cs

```diff
@@ -114,6 +114,12 @@ private static async Task Run(AppConfiguration configuration)
                 var productionUnitDetail = await cvrClient.GetProductionUnitDetailAsync(productionUnits[0].PNumber).ConfigureAwait(false);
 
                 Log.Information("Production Unit Detail data: {@ProductionUnitDetail}", productionUnitDetail);
+
+                Log.Information("Fetching company events using configuration {Name}", configuration.CvrNumber, cvrProvider.Name);
```

Comment:
Where are we using CVR number in the string?

Reply:
fixed